setup.py

```diff
@@ -35,6 +36,9 @@ def readfile(name):
     'hupper',
     ]
 
+if sys.version_info < (3, 2):
```

Comment:
This should be an `install_requires` tag instead, not a dynamic change, this way it ends up correctly in the wheels and things work as expected.